packages/transaction-controller/src/helpers/PendingTransactionTracker.ts

```diff
@@ -167,6 +170,22 @@ export class PendingTransactionTracker {
     }
   };
 
+  /**
+   * Adds a transaction to the polling mechanism for monitoring its status.
+   *
+   * This method forcefully adds a single transaction to the list of transactions
+   * being polled, ensuring that its status is checked, event emitted but no update is performed.
+   * It overrides the default behavior by prioritizing the given transaction for polling.
+   *
+   * @param transactionMeta - The transaction metadata to be added for polling.
+   *
+   * The transaction will now be monitored for updates, such as confirmation or failure.
+   */
+  addTransactionToPoll = (transactionMeta: TransactionMeta) => {
```

Comment:
Minor, can we avoid the assignment and define a standard method to be consistent?

Reply:
Done.